src/components/dayCard/DayCard.js

```diff
@@ -0,0 +1,19 @@
+function DayCard({ date, day, imgAltText, imgSrc, temperature, description }) {
+  const altText = imgAltText
+    ? imgAltText
+    : imgSrc?.substring(0, imgSrc.indexOf('.'));
+
+  return (
+    <div className='day-card'>
+      <h2>{day}</h2>
+      <p className='date'>{date}</p>
+      <img className='weather-icon' src={imgSrc} alt={altText} />
+      <p className='temperature'>
+        <b>{temperature} &deg;C</b>
```

Comment:
Can this (making the text bold) be a CSS property rather than a HTML tag?

Reply:
Done!